cmd/loopd/swapclient_server.go

```diff
@@ -342,6 +342,33 @@ func (s *swapClientServer) LoopIn(ctx context.Context,
 	}, nil
 }
 
+// GetLsatTokens returns all tokens that are contained in the LSAT token store.
+func (s *swapClientServer) GetLsatTokens(ctx context.Context,
+	_ *looprpc.TokensRequest) (*looprpc.TokensResponse, error) {
+
+	log.Infof("Get LSAT tokens request received")
```

Comment:
Nit: demote to debug?

Reply:
All the other RPCs have this as info so I think we can keep it.